### PR TITLE
Fixed missing API version query parameter in the request URL

### PR DESCRIPTION
**Pull request by jsorohova**
Closes: #1093

### CODE REVIEW

File: src/components/operations/operation-details/ko/runtime/operation-details.ts [hunk 1]
```diff
@@ -81,11 +81,19 @@ export class OperationDetails {
...
             if (api.type !== TypeOfApi.soap) {
                 operationPath += operation.displayUrlTemplate;
             }
 
+            if (api.apiVersionSet && api.apiVersion && api.apiVersionSet.versioningScheme === "Query") {
+                const separator = operationPath.indexOf("?") >= 0 ? "&" : "?";
+                versionPath = `${separator}${api.apiVersionSet.versionQueryName}=${api.apiVersion}`;
+            }
+            
+            operationPath += versionPath;
```
Comment (ygrik) [inline src/components/operations/operation-details/ko/runtime/operation-details.ts:95]: Please use existing Utils.addQueryParameter

File: src/components/operations/operation-details/ko/runtime/operation-details.ts [hunk 2]
```diff
@@ -86,11 +86,19 @@ export class OperationDetails {
...
+            let requestUrl = "";
+
             if (api.type === TypeOfApi.webSocket) {
-                return `${hostname}${Utils.ensureLeadingSlash(operationPath)}`;
+                requestUrl = `${hostname}${Utils.ensureLeadingSlash(operationPath)}`;
+            } else {
+                requestUrl = `https://${hostname}${Utils.ensureLeadingSlash(operationPath)}`;
+            }
+
+            if (api.apiVersionSet && api.apiVersion && api.apiVersionSet.versioningScheme === "Query") {
```
Comment (azaslonov) [inline src/components/operations/operation-details/ko/runtime/operation-details.ts:97]: nit: this
`api.apiVersionSet && api.apiVersion && api.apiVersionSet.versioningScheme === "Query"`
could be simplified a bit:
`api.apiVersion && api.apiVersionSet?.versioningScheme === "Query"`

File: src/components/operations/operation-details/ko/runtime/operation-details.ts [hunk 3]
```diff
@@ -86,11 +86,19 @@ export class OperationDetails {
                 operationPath += operation.displayUrlTemplate;
             }
 
+            let requestUrl = "";
+
             if (api.type === TypeOfApi.webSocket) {
```
Comment (azaslonov) [inline src/components/operations/operation-details/ko/runtime/operation-details.ts:91]: Not sure why we omit protocol for websockets here, there should be `ws` or `wss`. @ygrik can you please comment?

**Review by azaslonov — APPROVED** [review]

File: src/components/operations/operation-details/ko/runtime/operation-details.ts (hunk 3 above)
Comment (ygrik) [reply src/components/operations/operation-details/ko/runtime/operation-details.ts:91]: We show URL and protocols separately.
image

**Pull request merged**